w3af/plugins/crawl/open_api.py

```diff
@@ -448,10 +464,35 @@ def get_long_desc(self):
             * query_string_auth
             * header_auth
 
-        By default, the plugin validates Open API specification.
-        The validation may be disabled by 'no_spec_validation' configuration parameter.
+        By default, the plugin validates Open API specification. The validation
+        may be disabled by 'no_spec_validation' configuration parameter.
 
         During parsing an Open API specification, the plugin looks for parameters
-        which are passed to endpoints via HTTP headers, and enables them for further testing.
-        This behavior may be disabled by setting 'discover_fuzzable_headers' configuration parameter to False.
+        which are passed to endpoints via HTTP headers, and enables them
+        for further testing. This behavior may be disabled by setting
+        'discover_fuzzable_headers' configuration parameter to False.
+
+        The plugin tries to guess valid values for parameters of API endpoints
+        but the values highly depend on the context. If users have some knowledge
+        about correct values which may be used with the API endpoints, they can tell
+        the plugin about them via 'parameter_values_location' configuration parameter.
+        The option specifies a path to a YAML file, here is an example:
+
+            - path: /users/{user-id}
+              parameters:
+              - name: user-id
+                values:
+                - 1234567
+              - name: X-First-Name
+                values:
+                - John
+                - Bill
+            - path: /users
+              parameters:
+              - name: user-id
+                values:
+                - 1234567
+              - name: birth-date
+                values:
+                - 2000-01-02
```

Comment:
I believe that this feature requires its own documentation in https://github.com/andresriancho/w3af/blob/develop/doc/sphinx/scan-rest-apis.rst

I would make the example in `get_long_desc` shorter, and reference users to the documentation with something like: "For more information and examples about this feature read the framework documentation at https://docs.w3af.org"

Reply:
Okay, I'll update both description and docs. In fact, the docs need to be updated with descriptions of the recent changes/features. I hope I can find time to do that.